packages/phoenix-client/src/phoenix/client/resources/datasets/__init__.py

```diff
@@ -763,6 +763,7 @@ def add_examples_to_dataset(
         inputs: Iterable[Mapping[str, Any]] = (),
         outputs: Iterable[Mapping[str, Any]] = (),
         metadata: Iterable[Mapping[str, Any]] = (),
+        dataset_version_description: Optional[str] = None,
```

Comment:
This seems to be passed in as the overall dataset description below. For that reason I don't think this is the right param?

Note, I think given that we already have dataset in the method name, let's change this to 'version_description'